src/Quahog/Client.php

```diff
@@ -257,14 +261,34 @@ private function _sendCommand($command)
     /**
      * A wrapper to cleanly read a response from clamd.
      *
+     * @param bool $removeId
+     * @param string $readUntil
      * @return string
+     * @throws ConnectionException
      */
-    private function _receiveResponse()
+    private function _receiveResponse($removeId = false, $readUntil = "\n")
```

Comment:
Other than having to specify "\n" everyware bar one. Also only one place to change if it is ever switched to binary mode (eg commands terminated with a null byte instead of "\n")